lib/services_cli.rb

```diff
@@ -38,6 +38,10 @@ def user_path
       Pathname.new("#{ENV["HOME"]}/Library/LaunchAgents")
     end
 
+    def user_from_standard_plist_path
+      ServicesCli.user_path.to_path.match(%r{Users/(\w+)})[1]
```

Comment:
Perhaps this could be simplified to just use `ENV["HOME"]` directly?

Reply:
Much simpler! I was too far into "but how does this work and why is it weird?" last night, clearly. 😄